tesserocr.pyx

```diff
@@ -2069,6 +2069,11 @@ cdef class PyTessBaseAPI:
                 renderer = new TessOsdRenderer(outputbase)
                 return renderer
 
+        self._baseapi.GetBoolVariable("tessedit_create_alto", &b)
+        if b:
+            IF TESSERACT_VERSION >= 0x3999800:
+                renderer = new TessAltoRenderer(outputbase)
+
         self._baseapi.GetBoolVariable("tessedit_create_hocr", &b)
         if b:
             self._baseapi.GetBoolVariable("hocr_font_info", &font_info)
```

Comment:
here we would be overwriting the `renderer` so we should do a similar check to what we're doing later in the code:

```python
temp = new TessHOcrRenderer(outputbase, font_info)
if renderer == NULL:
    renderer = temp
else:
    renderer.insert(temp)
```

Reply:
Indeed, that slipped through. Done: 8f8b16c55743bae6a56fc851ab84071775df7b7a